documentation/configure.markdown

```diff
@@ -280,9 +280,11 @@ This controls how often, in minutes, to compact the database. The compaction pro
 
 ### `node-ttl`
 
-Auto-deactivate nodes that haven't seen any activity (no new catalogs,
-facts, etc) in the specified amount of time. You may specify the time
-as a string using any of the following suffixes:
+Mark as 'expired' nodes that haven't seen any activity (no new catalogs,
+facts, etc) in the specified amount of time. Expired nodes behave similarly
+to deactivated nodes; they are excluded from query results by default. 
```

Comment:
deactivated nodes aren't excluded on all endpoints (events for instance) so we might not want to make that claim